internal/runners/use/use.go

```diff
@@ -77,24 +77,19 @@ func (u *Use) Run(params *Params) error {
 
 	proj, err := project.FromNamespaceLocal(params.Namespace, u.config)
 	if err != nil {
-		if !project.IsLocalProjectDoesNotExist(err) {
-			return locale.WrapError(err, "err_use_project_frompath") // error reading from project file
+		if !project.IsLocalProjectDoesNotExistError(err) {
+			return locale.WrapError(err, "err_use", "Unable to use project")
 		} else if params.Namespace.Owner == "" {
-			projectsDir, err2 := storage.ProjectsDir(u.config)
-			if err2 != nil {
-				return locale.WrapError(err2, "err_use_cannot_determine_projects_dir", "") // this error takes precedence
-			}
-			errs.AddTips(err, locale.Tl("use_checkout_first", "", params.Namespace.Project))
-			projectDir := filepath.Join(projectsDir, params.Namespace.Project)
-			return locale.WrapInputError(err, "err_use_project_not_checked_out", "", params.Namespace.Project, projectDir)
+			// Note: use existing localized error message to workaround DX-740 for integration tests.
+			return locale.WrapInputError(err, "err_use_project_does_not_exist", err.Error())
```

Comment:
Nit: I'd prefer we drop the else on line 82, it makes for harder to read code and is unnecessary considering it returns on line 81.

Reply:
It's an else if. I want a fall through on else.

Reply:
My point is that if you drop the else (but keep the if) the logic is effectively the same but more readable.